packages/markup-include/src/CdcMarkupInclude.tsx

```diff
@@ -58,6 +58,12 @@ const CdcMarkupInclude: React.FC<CdcMarkupIncludeProps> = ({
   const { contentEditor, theme } = config || {}
   const data = configObj?.data
 
+  const variablesCheck = useMemo(() => {
```

Comment:
we'll need to find another way to handle this.

Reply:
The correct thing to do would be to search the data given the configuration you have for null values. It seems like you have a function below that does something like that, I wouldn't use that though. I think if you need to abstract some of the functions that are in that below function you should do that so they can be reused in both places.